config/field_test.yml

```diff
@@ -34,13 +49,11 @@ experiments:
       - 20
     goals:
       - user_creates_comment
-      - user_creates_comment_four_days_in_week
-      - user_views_article_four_days_in_week
-      - user_views_article_four_hours_in_day
-      - user_views_article_nine_days_in_two_week
-      - user_views_article_twelve_hours_in_five_days
-
-
+      - user_creates_comment_on_at_least_four_different_days_within_a_week
+      - user_views_pages_on_at_least_four_different_days_within_a_week
+      - user_views_pages_on_at_least_four_different_hours_within_a_day
+      - user_views_pages_on_at_least_nine_different_days_within_two_weeks
+      - user_views_pages_on_at_least_twelve_different_hours_within_five_days
```

Comment:
These are renamings of the original goals, rather than changes in the testing strategy - we're still getting the same comparison of results for this experiment as we collected in the last, just clarifying the names to be more specific?

The conditions in app/workers/users/record_field_test_event_worker.rb didn't change, just the observation labels.